bootstrap/index.ts

```diff
@@ -108,7 +117,8 @@ import "./arrayBuffer.js";
 
         nativeModuleProxy.DialogManagerAndroid.showAlert({
             title: "Error",
-            message: "Something went wrong while loading Aliucord! Please check the logs for more details.",
+            message: "Something went wrong while loading Aliucord! Please check the logs for more details."
```

Comment:
Since this captures all errors, theoretically this prompt can appear whenever
```suggestion
            message: "Something went wrong while running Aliucord! Please check the logs for more details."
```